frontend/public/components/utils/_log-window.scss

```diff
@@ -1,57 +1,33 @@
-$color-log-window-body-bg: $pf-color-black-1000;
 $color-log-window-header-bg: $pf-color-black-900;
 $color-log-window-text: $pf-color-black-150;
-$font-family-log-window-body: $font-family-monospace;
-$color-log-window-divider: $color-log-window-header-bg;
-
-.log-window {
-  color: $color-log-window-text;
-  background-color: $color-log-window-body-bg;
-
-  &--full-height {
-    height: 100%;
-    position: relative;
-  }
-}
 
 .log-window__header {
+  color: $color-log-window-text;
   background-color: $color-log-window-header-bg;
   padding: 8px 10px 5px 10px;
 }
 
-.log-window__body {
-  background-color: $color-log-window-body-bg;
-  font-family: $font-family-log-window-body;
-
-  &--full-height {
-    bottom: 0;
-    left: 0;
-    overflow: auto;
-    position: absolute;
-    right: 0;
-    top: 0;
+// The style of the buttons in dark theme log viewer is broken
+// this is a temp fix, could be removed if the style is fixed in the future version of PF log viewer
```

Comment:
Can we make sure there is a PF issue tracking and a comment linking to the issue?